src/audio_core/hle/aac_decoder.cpp

```diff
@@ -52,6 +52,29 @@ BinaryMessage AACDecoder::ProcessRequest(const BinaryMessage& request) {
     case DecoderCommand::Init: {
         BinaryMessage response = request;
         response.header.result = ResultStatus::Success;
+        if (decoder) {
+            NeAACDecClose(decoder);
+            decoder = nullptr;
+        }
+        decoderInitialized = false;
+
+        decoder = NeAACDecOpen();
```

Comment:
Can you move all this initialization to a AACDecoder::Init function, then we can remove all the code in the AACDecoder constructor. Afterall, there is no need to have the decoder ready until DecoderCommand::Init is received, right?